### ISSUE TITLE
evaluation  problem？

### ISSUE THREAD

**Issue by guyuchao** [ycszen/TorchSeg#78]
I find you do resize on the label to compute iou instead of resize your pred to origin label size? This may be due to iccurrate in iou because you loss details in small object.  
`label = cv2.resize(label,
                           (config.image_width // config.gt_down_sampling,
                            config.image_height // config.gt_down_sampling),
                           interpolation=cv2.INTER_NEAREST)`

**Comment by ycszen** [comment]
Yes. You can also change the evaluation code to the normal size with the same change to the training code.
